Assets/Scripts/Player/PlayerAttack.cs

```diff
@@ -29,7 +29,7 @@ private void TriggerAttack()
     {
         // TODO: Use object poll
         float direction = player.GetDirection();
-        projectile = Instantiate(projectilePrefab, transform.position, transform.rotation);
+        projectile = ObjectPool.instance.SpawnFromPool("bullets", gameObject.transform.position, gameObject.transform.rotation);
```

Comment:
Where are you creating this pool ? Or this is a quick test ?

Reply:
I guess I should watch the video :stuck_out_tongue:

Reply:
It's a sudo-singelton. So that's basically how you use it. You call the Class.instance. You need to create the pools in the inspection window in Unity.

Reply:
It's a sudo-singelton. So that's basically how you use it. You call the Class.instance. You need to create the pools in the inspection window in Unity.

Reply:
Ok, I see thanks :v: